numpy/lib/tests/test_histograms.py

```diff
@@ -40,20 +40,23 @@ def test_one_bin(self):
         assert_allclose(e, np.array([1., 2.]))
 
     def test_normed(self):
-        # Check that the integral of the density equals 1.
-        n = 100
-        v = np.random.rand(n)
-        a, b = histogram(v, normed=True)
-        area = np.sum(a * np.diff(b))
-        assert_almost_equal(area, 1)
-
-        # Check with non-constant bin widths (buggy but backwards
-        # compatible)
-        v = np.arange(10)
-        bins = [0, 1, 5, 9, 10]
-        a, b = histogram(v, bins, normed=True)
-        area = np.sum(a * np.diff(b))
-        assert_almost_equal(area, 1)
+        sup = suppress_warnings()
+        sup.filter(DeprecationWarning)
```

Comment:
Could check that the deprecation is actually recorded (twice), `log = sup.record(DeprecationWarning)`, and assert the length is 2. But not a big deal really. Possibly also check the density+normed given path just for principle.

Reply:
Length checked